src/Features/Core/Portable/ConvertToInterpolatedString/AbstractConvertRegularStringToInterpolatedStringRefactoringProvider.cs

```diff
@@ -19,14 +17,9 @@ namespace Microsoft.CodeAnalysis.ConvertToInterpolatedString
     /// <summary>
     /// Code refactoring that converts a regular string containing braces to an interpolated string
     /// </summary>
-    [ExportCodeRefactoringProvider(LanguageNames.CSharp, LanguageNames.VisualBasic, Name = PredefinedCodeRefactoringProviderNames.ConvertToInterpolatedString), Shared]
-    internal sealed class ConvertRegularStringToInterpolatedStringRefactoringProvider : CodeRefactoringProvider
```

Comment:
you can keep this original structure, but add a boolean to SyntaxFacts similar to:

```
        bool SupportsRecordStruct(ParseOptions options);
```

Reply:
I actually copied a bunch of code for this check from another interpolation string refactoring provider, e.g: https://github.com/dotnet/roslyn/blob/e396cd004bace8f23a1fd0da78f159c8e84b7b31/src/Features/VisualBasic/Portable/ConvertToInterpolatedString/VisualBasicConvertConcatenationToInterpolatedStringRefactoringProvider.vb#L21-L23
I didn't know, where I should put this check to utilize it. Thanks for `SyntaxFacts` suggestion!